source/source.go

```diff
@@ -174,7 +181,7 @@ func toRecords(kinRecords []types.Record, shardID string) []sdk.Record {
 				"shardId":        "kinesis-" + shardID,
 				"sequenceNumber": "kinesis-" + *rec.SequenceNumber,
 			},
-			sdk.RawData(kinPosBytes),
+			sdk.RawData(kinPos.SequenceNumber),
```

Comment:
Is the sequence number enough for the key? You are no longer using the shardID

Reply:
Actually it is not, yeah. I guess I was a bit uneasy having the position and the key be the same thing.